docs/client-sdk/client-sdk-porting-guide.md

```diff
@@ -93,6 +96,10 @@ set (BLOB_PATH .)
 set (TPM2_TCTI_TYPE tabrmd)
 set (RESALE true)
 set (REUSE true)
+
```

Comment:
the are more flags to add here can you compare with [this](https://github.com/fido-device-onboard/client-sdk-fidoiot/blob/master/cmake/cli_input.cmake)